settings/development/PandoraSettings_Streaming_Neutrino_DUNEFD.xml

```diff
@@ -0,0 +1,640 @@
+            </algorithm>
+            <algorithm type = "LArHitWidthClusterMerging"/>
+        </AlgorithmsShowerClustersV>
+    </algorithm>
+
+    <algorithm type = "LArClusteringParent">
+        <algorithm type = "LArTrackClusterCreation" description = "ClusterFormation"/>
+        <InputCaloHitListName>CaloHitListW</InputCaloHitListName>
+        <ClusterListName>ClustersW</ClusterListName>
+        <ReplaceCurrentCaloHitList>true</ReplaceCurrentCaloHitList>
+        <ReplaceCurrentClusterList>true</ReplaceCurrentClusterList>
+    </algorithm>
+    <algorithm type = "LArLayerSplitting"/>
+    <algorithm type = "LArDLTrackShowerStreamSelection">
+        <TrackListName>TrackClustersW</TrackListName>
+        <ShowerListName>ShowerClustersW</ShowerListName>
+    </algorithm>
+
+    <algorithm type = "LArStreaming">
+        <InputListNames>TrackClustersW ShowerClustersW</InputListNames>
+        <OutputListNames>TrackClustersW ShowerClustersW</OutputListNames>
+        <AlgorithmsTrackClustersW>
+            <algorithm type = "LArLongitudinalAssociation" />
+            <algorithm type = "LArTransverseAssociation" />
+            <algorithm type = "LArLongitudinalExtension" />
+            <algorithm type = "LArTransverseExtension" />
+            <algorithm type = "LArCrossGapsAssociation" />
+            <algorithm type = "LArCrossGapsExtension" />
+            <algorithm type = "LArOvershootSplitting" />
+            <algorithm type = "LArBranchSplitting" />
+            <algorithm type = "LArKinkSplitting" />
+            <algorithm type = "LArTrackConsolidation">
+                <algorithm type = "LArSimpleClusterCreation" description = "ClusterRebuilding"/>
+            </algorithm>
+            <algorithm type = "LArHitWidthClusterMerging"/>
+        </AlgorithmsTrackClustersW>
+        <AlgorithmsShowerClustersW>
+            <algorithm type = "LArLongitudinalAssociation" />
+            <algorithm type = "LArTransverseAssociation" />
+            <algorithm type = "LArLongitudinalExtension" />
+            <algorithm type = "LArTransverseExtension" />
+            <algorithm type = "LArCrossGapsAssociation" />
+            <algorithm type = "LArCrossGapsExtension" />
+            <algorithm type = "LArOvershootSplitting" />
+            <algorithm type = "LArBranchSplitting" />
+            <algorithm type = "LArKinkSplitting" />
+            <algorithm type = "LArTrackConsolidation">
+                <algorithm type = "LArSimpleClusterCreation" description = "ClusterRebuilding"/>
+            </algorithm>
+            <algorithm type = "LArHitWidthClusterMerging"/>
+        </AlgorithmsShowerClustersW>
+    </algorithm>
+
+    <algorithm type = "LArListDeletion">
+        <ClusterListNames>ClustersU ClustersV ClustersW</ClusterListNames>
+        <VertexListNames>CandidateVertices3D NeutrinoVertices3D</VertexListNames>
+    </algorithm>
+    <algorithm type = "LArListMerging">
```

Comment:
So origins of clusters effectively lost here (but can presumably still query cluster for its preliminary particle id?). Two instances of this alg needed? (would need to double check details).

**Is it just better to edit the interface for the vertexing alg? Keep track vs. shower nature as that used within the context of the vertexing alg.**

Reply:
Long term I think we want to update the vertexing to be able to work directly with the streams.